package.json

```diff
@@ -64,5 +64,8 @@
       "src/*.js",
       "!src/index.js"
     ]
+  },
+  "dependencies": {
+    "babel-core": "^6.26.3"
```

Comment:
We don't need this dependency. Please remove it. 

If you need `babel-core` to build and test locally you can add it to the `devDependencies`.

Reply:
Removed dependency